core/storage/collection/gae_models.py

```diff
@@ -32,12 +32,26 @@
 
 class CollectionSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
     """Storage model for the metadata for a collection snapshot."""
-    pass
+
+    @staticmethod
+    def get_export_policy():
+        """Model relates to version history of entity. Not included in export
```

Comment:
I think you need to fix these docstrings in order to avoid implying that they don't need to be implemented. 

Ditto elsewhere for all the other models with a TO_BE_IMPLEMENTED policy.

Reply:
Done